test/runtime/Policies.go

```diff
@@ -868,7 +836,15 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 					"%s"
 				]
 			}]
-		}]`, httpd1Label, httpd2Label, ipv4OtherNet)
+		},
+		{
+			"endpointSelector": {"matchLabels":{"%s":""}},
+			"egress": [{
+				"toEndpoints": [
+					{"matchLabels":{"%s":""}}
+				]
+			}]
+		}]`, httpd1Label, httpd2Label, ipv4OtherNet, "id.app3", httpd1Label)
```

Comment:
- Could you change httpd1Label to use only one time, and used `%[1]s` instead of `%s`
- Could you set "id.app3" as a const, want to avoid the hardcoding in the code. Should be something like this: 

```
script = fmt.Sprintf(`
		[{
			"endpointSelector": {"matchLabels":{"%[1]s":""}},
			"ingress": [{
				"fromEndpoints": [
					{"matchLabels":{"%[2]s":""}}
				]
			}, {
				"fromCIDR": [
					"%[3]s"
				]
			}]
		},
		{
			"endpointSelector": {"matchLabels":{"%[4]s":""}},
			"egress": [{
				"toEndpoints": [
					{"matchLabels":{"%[1]s":""}}
				]
			}]
		}]`, httpd1Label, httpd2Label, ipv4OtherNet, "id.app3")

```